config.yml

```diff
@@ -127,3 +128,4 @@ desktop:
   data_prep_outputs_path: "./outputs"
   asset_impact_data_path: "./ai_inputs"
   factset_data_path: "./factset_inputs"
+  export_archives: FALSE
```

Comment:
```suggestion
```
While I appreciate trying to maintain "desktop" behavior, I'd like it to remain as close to "default" as reasonable.